toothpick-runtime/src/main/java/toothpick/ToothPick.java

```diff
@@ -100,11 +107,27 @@ public static void inject(Object obj, Scope scope) {
     injector.inject(obj, scope);
   }
 
-  // Not synchronized, called by closeScope that is synchronized
```

Comment:
You should leave this comment or add something like:
This method is not thread safe.